frame/vamm/src/lib.rs

```diff
@@ -984,7 +988,7 @@ pub mod pallet {
 
 			let price = Self::u256_to_balance(price_u256)?;
 
-			Ok(DecimalOf::<T>::from_inner(price))
+			Ok(price.into_decimal()?)
```

Comment:
Cool, I didn't predict it would work for unsigned decimals right off the bat

Reply:
Yeah I thought the same, thankfully it just works :tm: